prepare-videochat.sh

```diff
@@ -338,7 +347,7 @@ fi
 if lsmod | grep v4l2loopback >/dev/null 2>/dev/null; then
     # module is already loaded, do nothing
     :
-else
+elif [ $CAPTURE_STREAM = v -o $CAPTURE_STREAM = av ]; then
```

Comment:
Yes, but audio+video should be the default - it feels as if now the default is only audio. Otherwise we'll get complaints from people using the script for the first time.